docs/imviz/plugins.rst

```diff
@@ -397,6 +397,8 @@ are not stored. To save the current result before submitting a new query, you ca
     The table returned from the API above may cover more sources than shown in the currently zoomed-in
     portion of the image. Additional steps will be needed to filter out these points, if necessary.
 
+After a search is performed, a table is populated on the right that contains the
+right ascension, declination, and the object ID of the found sources. 
```

Comment:
```suggestion
Performing a search populates a table that contains the
right ascension, declination, and the object ID of the found sources. 
```
I think this is a little less passive-voice. Also I wasn't clear what the table is on the right of (it seems like it's just at the bottom of the plugin) so I deleted that phrase.